package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "commander": "^2.8.1",
+    "ether-pudding": "https://github.com/VoR0220/ether-pudding.git",
```

Comment:
I agree it should but there are currently some issues that we run into when we utilize ether-pudding because the babel version is not specified. Truffle gets around this by specifying which version of babel, but because Embark does not naturally have babel, it automatically updates to the latest version which throws a weird node.js error forcing you to download the Core package...there's also a weird Pudding Version error involved and I've made note of this to @tcoulter